securedrop/.rsync-filter

```diff
@@ -1,4 +1,10 @@
 exclude config.py
+exclude *.pyc
```

Comment:
This should not be necessary because everything is explicitly included. Why do you think it is?